src/main/java/com/google/api/codegen/transformer/ModelTypeNameConverter.java

```diff
@@ -14,10 +14,9 @@
  */
 package com.google.api.codegen.transformer;
 
-import com.google.api.codegen.config.FieldConfig;
-import com.google.api.codegen.config.FieldModel;
-import com.google.api.codegen.config.InterfaceModel;
-import com.google.api.codegen.config.ProtoInterfaceModel;
+import static com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Type.TYPE_STRING;
```

Comment:
Probably shouldn't import the value itself, but just `Type`. Or something lower in the chain.

Reply:
done.